source/code/plugins/oms_omi_lib.rb

```diff
@@ -73,7 +73,13 @@ def get_cim_to_oms_mappings(cimproperties)
     cimproperties.each { |maps|
       cim_name = maps["CimPropertyName"]
       oms_name = maps["CounterName"]
-      cim_to_oms[cim_name] = oms_name
+      # Map cim_name to both CounterName and DisplayName
+      if maps.has_key?("DisplayName")
+        display_counter_name = maps["DisplayName"]
+        cim_to_oms[cim_name] = [oms_name, display_counter_name]
```

Comment:
I don't like that the cim_to_oms hash now has strings and arrays as values and we use the type to determine the behavior.